core/src/db/DB.h

```diff
@@ -11,6 +11,7 @@
 
 #pragma once
 
+#include <map>
```

Comment:
We'd better create new files for DB.h / DBImpl.h / DBImpl.cpp

Reply:
This PR is a placeholder for CI health check. The integration approach is open and will be discussed after the holiday.